### PR TITLE
Cong-Ruby

### PR DESCRIPTION
**Pull request by congpine**

### CODE REVIEW

File: week0001/counter.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+p "Start counting:" 
+for counter in 0..100
+  if counter % 3 == 0
+    p "Fizzi" 
+  elsif counter % 5 == 0
```
Comment (vietlq) [inline week0001/counter.rb]: This condition is not right ;) Please check the requirements again.

File: week0001/counter.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+
+p "Start counting:" 
+for counter in 0..100
+  if counter % 3 == 0
+    p "Fizzi"
```
Comment (vietlq) [inline week0001/counter.rb]: Fizz, not Fizzi

File: week0001/counter.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+
+p "Start counting:" 
+for counter in 0..100
+  if counter % 3 == 0
+    p "Fizzi" 
+  elsif counter % 5 == 0 
+    p "Buzz" 
+  else
+    p counter
```
Comment (vietlq) [inline week0001/counter.rb]: There was no requirement to print the counter

File: week0001/counter.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+
+p "Start counting:"
```
Comment (vietlq) [inline week0001/counter.rb]: No such requirement to print this line. Also rename your file like this: `fizz_buzz_congpine.rb`. Thanks!

File: week0001/counter.rb [hunk 5]
```diff
@@ -0,0 +1,11 @@
+
+p "Start counting:" 
+for counter in 0..100
```
Comment (vietlq) [inline week0001/counter.rb]: The requirement was from 1 to 100 ;)

**Comment by congpine** [comment]
hi anh 
I have fixed it. please review

File: week0001/fizz_buzz_congpine.rb [hunk 6]
```diff
@@ -0,0 +1,11 @@
+for counter in 1..100
+
+  if counter % 3 == 0
+    p "Fizz"
```
Comment (vietlq) [inline week0001/fizz_buzz_congpine.rb]: Use `puts` not `p` because `p` wraps your string in double quotes and hence produces wrong output!

Comment (congpine) [inline week0001/fizz_buzz_congpine.rb]: hmm, i though the requirement was: `prints the string "Fizz" `

**Pull request merged**